core/bootstrap_espresso.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+use EventEspresso\core\exceptions\InvalidClassException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\exceptions\InvalidIdentifierException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\container\exceptions\ServiceExistsException;
+use EventEspresso\core\services\container\exceptions\ServiceNotFoundException;
```

Comment:
👍🏻 

this should be fine... 
we couldn't do this before here because there were still a few users on servers running PHP 5.2 😕 
and this originally was not behind a php version check

Reply:
i call those days the "dark ages"